modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/init.jsp

```diff
@@ -164,6 +164,7 @@ if (ddmDisplayContext != null) {
 	scopeClassNameId = PortalUtil.getClassNameId(ddmDisplay.getStructureType());
 	scopeStorageType = ddmDisplay.getStorageType();
 	scopeTemplateType = ddmDisplay.getTemplateType();
+	scopeTitle = ddmDisplayContext.getScopedStructureLabel();
```

Comment:
Hey @naraandrade, Have you checked the possible impacts on components other than Kaleo Forms? I'm question this because we are overriding the value that comes from the URL(`String scopeTitle = ParamUtil.getString(request, "scopeTitle");`).  A safe fix would be  retrieve the value from `ddmDisplayContext` when the URL value is empty only. What do you think?.

Thx

Reply:
I can do it if you want or do not have time. :)

Reply:
@rafaprax "A safe fix would be retrieve the value from ddmDisplayContext when the URL value is empty only" This is exactly what is done inside the method. So, in the worst case, the override would put the same value that was already defined for the variable

Thx

Reply:
@naraandrade So. it's ok then. :)